### PR TITLE
KIALI-698 Remove prettier from yarn start, yarn test and yarn build

### PR DESCRIPTION
**Pull request by israel-hdez**
Prettier fixes code syntax in order to comply with _good_ code stylings.
Good syntax is only relevant for developers/humans. So, it's not really
needed when running `yarn start`, `yarn test` nor `yarn build`, which
are meant to build the code (good syntax is irrelevant for
compilers/transpilers).

In CI, prettier is already called directly to trigger a failure if PRs
or code being merged/pushed. Removing prettier from build and test
stages will have the benefit that prettier will be called once in CI.

In dev envs, prettier is part of `yarn precommit` which can be
configured to interrupt a commit if prettier finds and fixes syntax
issues, allowing early detection before pushing to the repo.

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#125](https://paste.fedoraproject.org/paste/c2Hw-Pg1QHncygmCLV-Ojg)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1421]](https://paste.fedoraproject.org/paste/cHEoe56BZpmkSCTiKP8IgA)
  * :heavy_check_mark: Istio-Upstream [#[294]](https://paste.fedoraproject.org/paste/GTHjj-tVDz0qQrXg1GvmdQ)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1384, 1385]](https://paste.fedoraproject.org/paste/tQJiisBB~4~IfKdJiIYKHg)
  * :heavy_check_mark: install-bookinfo [#[782]](https://paste.fedoraproject.org/paste/PrAFnWQX4NZ-T~A83LKYGA)
  * :heavy_check_mark: Kiali-on-Openshift [#[784]](https://paste.fedoraproject.org/paste/jX4uL3gmMKMdg6flHW4bHw)
  * :heavy_check_mark: run-kiali-e2e-tests [#[229]](https://paste.fedoraproject.org/paste/waYXtrxIC8Nnzkd5RG8AGA)


KIALI_NAME=kialiuipr716-istio-system

**Comment by josejulio** [comment]
> (good syntax is irrelevant for
> compilers/transpilers).

I think that it is still relevant

```bash
echo " " >> src/index.tsx
yarn start
```

And you will get:
```
Failed to compile.                                                                                                                                                              
                                                                                                                                                                            
/home/josejulio/Documents/redhat/kiali/kiali-ui/src/index.tsx                                                                                                                   
(8,1): Consecutive blank lines are forbidden                                                                                                                                    
```

see: https://asciinema.org/a/rE9OrYSW8lwIqBicBq8x2zVty

WDYT?

File: package.json [hunk 1]
```diff
@@ -128,5 +128,6 @@
     "node": ">=8.0.0 ",
     "npm": ">=5.0.0 ",
     "yarn": ">=1.0.0 "
-  }
+  },
+  "proxy": "https://kiali-istio-system.192.168.99.100.nip.io/"
```
Comment (jotak) [inline package.json]: proxy line is there by mistake I guess?

Comment (israel-hdez) [reply package.json]: Oops!! Yes, it's a mistake. Fixed!

**Comment by israel-hdez** [comment]
@josejulio 
> I think that it is still relevant

Hmm... AFAIK, that's tslint, not the typescript compiler. Again, it's only to promote good coding.
Looks like you can disable it: https://stackoverflow.com/questions/41952766/is-it-possible-to-disable-tslint-in-tslint-json

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#126](https://paste.fedoraproject.org/paste/Gckn5wMrFsoogXexEG-tPw)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1425]](https://paste.fedoraproject.org/paste/FrADdlQusketrMwjX31tDQ)
  * :heavy_check_mark: Istio-Upstream [#[297]](https://paste.fedoraproject.org/paste/4iM-Bwyv-d6tj66mzbNy0g)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1395, 1396]](https://paste.fedoraproject.org/paste/52AiEa10wc~3wcOooH6EBw)
  * :heavy_check_mark: install-bookinfo [#[789]](https://paste.fedoraproject.org/paste/RMb5b1~i30icbD5AYRjkeg)
  * :heavy_check_mark: Kiali-on-Openshift [#[790]](https://paste.fedoraproject.org/paste/BnyWfPtDPknDP1IDsKbn2w)
  * :heavy_check_mark: run-kiali-e2e-tests [#[235]](https://paste.fedoraproject.org/paste/B5yQjmZTkqvfKDU0yCeZTw)


KIALI_NAME=kialiuipr716-istio-system

**Comment by josejulio** [comment]
> Hmm... AFAIK, that's tslint, not the typescript compiler. Again, it's only to promote good coding.
> Looks like you can disable it: https://stackoverflow.com/questions/41952766/is-it-possible-to-disable-tslint-in-tslint-json

Right, I just want to make sure we understand that by disabling this prettier, we are going to get these errors more frequent. To be honest I always hate to have to follow the tslint when i want to do a quick test e.g. comment a line of code and the linter starts complaining "Unused variable", need to comment that variable (which uses a type) "Unused import", need to comment the import... etc.

I suggest that if we are removing prettier from yarn start,  disable the tslint for that too...

**Comment by mtho11** [comment]
Let's give this a try and see how we like it. We can change it later if we don't like it.
@israel-hdez please send a message to dev-list regarding this change

**Comment by mtho11** [comment]
@josejulio regarding tslint let's do that in different PR and wait and see how we like this first.

**Pull request merged**